contracts/Registry/LockableItemRegistry.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+import './OwnedItemRegistry.sol';
+
+// provides a mapping of unlock times for items. Only allows
+// item removal when the unlock time has been exceeded.
+contract LockableItemRegistry is OwnedItemRegistry {
```

Comment:
why does this need `OwnedItemRegistry` logic? not seeing it used anywhere.

Also I'm thinking something like `TimelockableItemRegistry` is a better name. Lockable sounds more general like you can lock/unlock items at more at will.

Reply:
@emaG3m you're right, it doesn't need to extend this. Fixed that. Also changed the name to `TimelockableItemRegistry`